applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -164,6 +169,40 @@ handle_participant_uri(?HTTP_POST, Context, ConfId, _ParticipantId) ->
 %%% REST API Actions
 %%%===================================================================
 
+-spec find_schema(ne_binary()) -> api_object().
+find_schema(<<_/binary>> = Schema) ->
+    case kz_json_schema:load(Schema) of
+        {'ok', SchemaJObj} -> SchemaJObj;
+        {'error', _E} ->
+            lager:error("failed to find schema ~s: ~p", [Schema, _E]),
+            'undefined'
+    end.
+
+save(Context, JObj) ->
+    JObj0 = crossbar_doc:update_pvt_parameters(JObj, Context),
+    case kz_datamgr:save_doc(cb_context:account_db(Context), JObj0) of
+        {'error', Error} ->
+            crossbar_doc:handle_datamgr_errors(Error, kz_doc:id(JObj0), Context);
+        {'ok', JObj1} ->
+            crossbar_doc:handle_datamgr_success(JObj1, Context)
+    end.
+
+create_conference(JObj) ->
+    SchemaJObj = find_schema(<<"conferences">>),
```

Comment:
Can't you use `Ctx = cb_context:validate_request_data(<<"conferences">>, Context)` ?